configs/getConfig.js

```diff
@@ -0,0 +1,20 @@
+const _ = require("lodash");
```

Comment:
i'm okay using ES6 where we can, but in this case, we do need a deep merge, so lodash.merge makes sense to me